src/libmongoc/tests/test-mongoc-sdam-monitoring.c

```diff
@@ -699,11 +699,11 @@ _test_heartbeat_events (bool pooled, bool succeeded)
    start = bson_get_monotonic_time ();
 
    if (pooled) {
-      pool = test_framework_client_pool_new_from_uri (uri, NULL);
+      pool = mongoc_client_pool_new (uri);
```

Comment:
Please update this file to use test_framework_client_pool_new_from_uri and replace the calls to mock_server_receives_legacy_hello with mock_server_receives_any_hello.